windows/scripts/extract_kernels.py

```diff
@@ -124,7 +99,6 @@ def cleanup(self, warn=False):
             if warn:
                 print('Implicitly cleaning up {!r}'.format(self))
 
-
```

Comment:
x2 blank lines between Python functions (except if they are class methods, in which case it's only a single line).